project.clj

```diff
@@ -2,6 +2,13 @@
 
 (defproject jme-clj "0.1.13"
 
+  :codox {:metadata    {:doc        "**TODO**: write docs"
+                        :doc/format :markdown}
+          :output-path "docs"
+          ;;:source-uri  "https://github.com/simon-brooke/jme-clj/blob/main/{filepath}#L{line}"
+          :source-uri   "file:///home/simon/workspace/jme-clj/{filepath}#L{line}"
```

Comment:
We'd like to change the personal path :)

Reply:
Oooops! Sorry, yes, of course!